.github/workflows/puppeteer-macos.yml

```diff
@@ -0,0 +1,130 @@
+name: Puppeteer MacOS
+
+##########################################################################
+
+env:
+  JAVA_OPTS: "-Xms512m -Xmx6048m -Xss128m -XX:ReservedCodeCacheSize=512m -server"
+  GRADLE_OPTS: "-Xms512m -Xmx6048m -Xss128m -XX:ReservedCodeCacheSize=512m -server"
+  GRADLE_BUILDCACHE_USER: ci
+  GRADLE_BUILDCACHE_PSW: ${{ secrets.GRADLE_BUILDCACHE_PSW }}
+  DUO_SECURITY_ADMIN_SKEY: ${{ secrets.DUO_SECURITY_ADMIN_SKEY }}
+  DUO_SECURITY_ADMIN_IKEY: ${{ secrets.DUO_SECURITY_ADMIN_IKEY }}
+  DUO_SECURITY_UNIVERSAL_PROMPT_SKEY: ${{ secrets.DUO_SECURITY_UNIVERSAL_PROMPT_SKEY }}
+  DUO_SECURITY_UNIVERSAL_PROMPT_IKEY: ${{ secrets.DUO_SECURITY_UNIVERSAL_PROMPT_IKEY }}
+  DUO_SECURITY_WEBSDK_SKEY: ${{ secrets.DUO_SECURITY_WEBSDK_SKEY }}
+  DUO_SECURITY_WEBSDK_IKEY: ${{ secrets.DUO_SECURITY_WEBSDK_IKEY }}
+  DUO_SECURITY_WEBSDK_AKEY: ${{ secrets.DUO_SECURITY_WEBSDK_AKEY }}
+  DUO_SECURITY_API_HOST: ${{ secrets.DUO_SECURITY_API_HOST }}
+  IMGUR_CLIENT_ID: ${{ secrets.IMGUR_CLIENT_ID }}
+  TERM: xterm-256color
+  JDK_CURRENT: 11
+  NODE_CURRENT: '16'
+  SCENARIO_REGEX: ".*" # Use this to limit which tests run
+
+on:
```

Comment:
I think it might be better to run this "alternative" setups on a schedule; sort of a compromise between running them all the time and never.

Reply:
Yes, a schedule would work. I want to look into how it could avoid running on forks if someone enables pipelines on their forks... I wouldn't want it kicking off all those jobs every week without someone noticing.  It would be nice if the schedule only ran on certain branches but I didn't see anywhere to do that, but a rule could probably limit that.